### PR TITLE
Cockpit 196

### PR DESCRIPTION
**Pull request by mvollmer**
Rendered: https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-196.html

### CODE REVIEW

File: _posts/2019-06-12-cockpit-196.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+slug: cockpit-196
+category: release
+summary: Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 196.
+
+### Machines: Virtual machines can be installed directly from a URL
```
Comment (martinpitt) [inline _posts/2019-06-12-cockpit-196.md]: URLs already worked for Fedora/RHEL package trees, the new thing is that it now also works for *.iso. The text below clarifies this, but it's a bit ambiguous here.

Comment (mvollmer) [reply _posts/2019-06-12-cockpit-196.md]: I have changed the title.

**Comment by garrett** [comment]
My one optional, nitpicky suggestion is to reword the sentence "It is now possible to specify a URL of a ISO file when creating a virtual machine" to be a little less stuffy and jargony.

Perhaps:
"Cockpit can now create a virtual machine from a ISO disc image URL."

**Comment by mvollmer** [comment]
> "Cockpit can now create a virtual machine from a ISO disc image URL."

Thanks, I used that.

**Pull request merged**